diff --git i/package.json w/package.json
index 982f11848..893a706e1 100644
--- i/package.json
+++ w/package.json
@@ -3,11 +3,11 @@
     "node": ">=12"
   },
   "name": "ledger-live-desktop",
-  "productName": "Ledger Live",
-  "description": "Ledger Live - Desktop",
+  "productName": "Ledger Live Nightly",
+  "description": "Ledger Live - Desktop Nightly",
   "version": "2.32.2",
   "author": "Ledger Live Team <team-live@ledger.fr>",
-  "repository": "https://github.com/LedgerHQ/ledger-live-desktop",
+  "repository": "https://github.com/LedgerHQ/ledger-live-desktop-canary",
   "license": "MIT",
   "private": true,
   "main": "./.webpack/main.bundle.js",
diff --git i/src/main/updater/init.js w/src/main/updater/init.js
index 52e06e733..21b02301f 100644
--- i/src/main/updater/init.js
+++ w/src/main/updater/init.js
@@ -22,8 +22,9 @@ const sendStatus = (status: UpdateStatus, payload?: *) => {
 const handleDownload = async info => {
   try {
     sendStatus("checking");
-    const appUpdater = await createElectronAppUpdater({ feedURL: UPDATE_CHECK_FEED, info });
-    await appUpdater.verify();
+    await createElectronAppUpdater({ feedURL: UPDATE_CHECK_FEED, info });
+    // const appUpdater = await createElectronAppUpdater({ feedURL: UPDATE_CHECK_FEED, info });
+    // await appUpdater.verify();
     sendStatus("check-success");
   } catch (err) {
     logger.critical(err);
diff --git i/src/renderer/components/Updater/Banner.js w/src/renderer/components/Updater/Banner.js
index 11a0042a9..61bfe9940 100644
--- i/src/renderer/components/Updater/Banner.js
+++ w/src/renderer/components/Updater/Banner.js
@@ -95,7 +95,7 @@ const UpdaterTopBanner = () => {

     if (!VISIBLE_STATUS.includes(status)) return null;

-    let content: ?Content = CONTENT_BY_STATUS(
+    const content: ?Content = CONTENT_BY_STATUS(
       quitAndInstall,
       reDownload,
       downloadProgress,
@@ -105,17 +105,17 @@ const UpdaterTopBanner = () => {

     if (!content) return null;

-    if (__APP_VERSION__.includes("nightly")) {
-      content = {
-        Icon: IconInfoCircle,
-        message: <Trans i18nKey="update.nightlyWarning" />,
-        right: (
-          <FakeLink onClick={() => openURL(urls.liveHome)}>
-            <Trans i18nKey="update.downloadNow" />
-          </FakeLink>
-        ),
-      };
-    }
+    // if (__APP_VERSION__.includes("nightly")) {
+    //   content = {
+    //     Icon: IconInfoCircle,
+    //     message: <Trans i18nKey="update.nightlyWarning" />,
+    //     right: (
+    //       <FakeLink onClick={() => openURL(urls.liveHome)}>
+    //         <Trans i18nKey="update.downloadNow" />
+    //       </FakeLink>
+    //     ),
+    //   };
+    // }

     return <TopBanner id={"app-update-banner"} content={content} status={status} />;
   }
diff --git i/ui-lib/package.json w/ui-lib/package.json
index 0d456b5eb..8d299f5b5 100644
--- i/ui-lib/package.json
+++ w/ui-lib/package.json
@@ -7,7 +7,7 @@
   "description": "Ledger Live - UI Desktop",
   "version": "0.0.1",
   "author": "Ledger Live Team <team-live@ledger.fr>",
-  "repository": "https://github.com/LedgerHQ/ledger-live-desktop/ui-lib",
+  "repository": "https://github.com/LedgerHQ/ledger-live-desktop-canary/ui-lib",
   "license": "MIT",
   "private": true,
   "main": "./index.js",
